### ISSUE TITLE
something related to bitcoin?

### ISSUE THREAD

**Issue by guaka** [libreprojects/libreprojects#89]
#35, Bitcoin is not hosted, but somehow it would be nice to include something related to Bitcoin.

AFAIK https://blockchain.info/ is all open source.
There are surely more but before diving deeper into this I want to know if there's a chance this stuff will be added :)

**Comment by jancborchardt** [comment]
Yeah, probably. I have some doubts about Bitcoin but that’s a different story.

What would be neat is a platform to get started with Bitcoin. I think that would need to be a hosted wallet service. And Blockchain seems to be that, although not _all_ open source: http://bitcoin.stackexchange.com/questions/4438/is-blockchain-info-open-source – is this still current, or can you effectively host your own wallet with that code?

What do others think? @libreprojects/contributors

**Comment by jancborchardt** [comment]
(Sorry, hit the close button by accident. ;)

**Comment by guaka** [comment]
There are some other projects that fit the spirit more but I'm not aware of any easy wallets out there. I think [Mastercoin's omniwallet](https://github.com/mastercoin-MSC/omniwallet) could be that very soon.

Projects to keep an eye on:
basic income: http://ucoin.io/  http://openudc.org/
Gesell-style "freedom from usury": http://freico.in/
coins for citizens: http://auroracoin.org/
